snapshot_manager/snapshot_manager/util.py

```diff
@@ -693,3 +704,46 @@ def serialize_config_map_to_github_matrix(
             res["name"].append(strat)
 
     return json.dumps(res)
+
+
+def sanitize_uuid(id: str | uuid.UUID | None) -> uuid.UUID:
```

Comment:
This function is technically not new. It was moved from `tfutil.sanitize_request_id` here and `tfutil.sanitize_request_id` now calls this function.